backend/core/src/migration/1625825502154-refactor-unit-types-rent-types-and-priority-types.ts

```diff
@@ -12,7 +12,7 @@ export class refactorUnitTypesRentTypesAndPriorityTypes1625825502154 implements
       `ALTER TABLE "units" ADD CONSTRAINT "FK_1e193f5ffdda908517e47d4e021" FOREIGN KEY ("unit_type_id") REFERENCES "unit_types"("id") ON DELETE NO ACTION ON UPDATE NO ACTION`
     )
 
-    const unitTypeSeeds = ["studio", "oneBdrm", "twoBdrm", "threeBdrm", "fourBdrm"]
+    const unitTypeSeeds = ["studio", "oneBdrm", "twoBdrm", "threeBdrm", "fourBdrm", "fiveBdrm"]
```

Comment:
If we're planning to submit this upstream you'd probably need to generate a new migration, no?

Reply:
Yeah I think so. (My mental model for how it works when there's a new migration: only the new migrations are applied to the deployed DB, there's never a `yarn db:reseed`-like step that re-applies all migrations for the deployed DB.)

Reply:
The bonus of dropping it is we won't have to deal with this migration!

Reply:
Sounds good :)